src/components/TrackTile/TrackTile.tsx

```diff
@@ -28,11 +28,13 @@ export function TrackTile({
   hideSettingsButton = false,
   title = '',
   styles: instanceStyles,
+  shouldUseOntology,
 }: TrackTileProps) {
   const { styles } = useStyles(defaultStyles, instanceStyles);
   const valuesContext: TrackerValuesContext = {
     system: TRACKER_CODE_SYSTEM,
     codeBelow: TRACKER_CODE,
+    shouldUseOntology: shouldUseOntology,
```

Comment:
Do we need this?

Reply:
Yea, I should've called it out - can't remember if I purposefully pushed this to the next PR or I missed it in the last, but without this trackers using advanced screen ([new feature](https://github.com/lifeomic/react-native-sdk/pull/298/files)) wouldn't do the required ontology fetch to take advantage of value roll-up logic.

Reply:
Got it! Sorry I forgot about that change. Thanks for the clarification! https://github.com/lifeomic/react-native-sdk/pull/310/files#diff-3071873bc2ad5c60590af1ec4dea23282c814172d69c34c26f8a5318b3153b0eR98